languages/csharp/extensions/int.tmt

```diff
@@ -0,0 +1,6 @@
+~[EXPR_int]~ -> 0
+~[EXPR_int]~ -> -1
+~[EXPR_int]~ -> 42
+~[EXPR_int]~ -> 1000
```

Comment:
I think such rules are not needed

Reply:
They are present in the Java templates, so I kept them here. Maybe @DaniilStepanov can better explain the idea

Reply:
Ok, let's kept. I think this is can't ruin fuzzing